nilearn/tests/test_signal.py

```diff
@@ -366,7 +367,23 @@ def test_clean_confounds():
                                    np.zeros((20, 2)))
 
 
+def test_clean_img():
+
```

Comment:
Any special reason to do that? We don't need strong repeatbility on this one. Also, we never use this in the rest of the tests.